<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filtros</class>
 <widget class="QDialog" name="Filtros">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filtrar Operaciones</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxFiltros">
     <property name="title">
      <string>Criterios de Filtrado</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTipo">
        <property name="text">
         <string>Tipo de Operaci√≥n:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboTipo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFechaInicio">
        <property name="text">
         <string>Fecha Inicio:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEditInicio"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFechaFin">
        <property name="text">
         <string>Fecha Fin:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEditFin"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPlaza">
        <property name="text">
         <string>Plaza:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboPlaza"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnFiltrar">
       <property name="text">
        <string>Filtrar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>